src/App.js

```diff
@@ -1,33 +1,61 @@
-import React from 'react';
 import './App.css';
+import BodyComponent from './components/BodyComponent';
 import Header from './components/Header';
-import Heading from './components/Heading';
+import {createBrowserRouter, RouterProvider, Outlet} from 'react-router-dom'
+import ErrorComponent from './components/ErrorComponent';
+import TeamMemberDetails from './components/TeamMemberDetails';
+//import About from './components/About';
+import ProfileComponent from './components/ProfileComponent'
+import NestedProfile from './components/NestedProfile';
+import { lazy, Suspense } from 'react';
```

Comment:
library imports should be at top. and can give an empty line between library imports and local file imports

Reply:
yeah, hv checked that in assignment 10